pyrate/core/config.py

```diff
@@ -639,6 +718,22 @@ def coherence_paths_for(path: str, params: dict, tif=False) -> str:
 }
 """dict: basic validation functions for coherence parameters."""
 
+_BASELINE_VALIDATION = {
+    BAS_FILE_LIST: (
+        lambda a: a is not None and not os.path.exists(a),
+        f"'{BAS_FILE_LIST}': if file is provided it must exist."
+    ),
+}
+"""dict: basic validation functions for baseline parameters."""
+
+_LOOKUPTABLE_VALIDATION = {
+    LT_FILE: (
+        lambda a: a is not None and not os.path.exists(a),
+        f"'{LT_FILE}': if file is provided it must exist."
+    ),
+}
+"""dict: basic validation functions for baseline parameters."""
```

Comment:
`baseline` should be `lookuptable`

Reply:
see https://github.com/GeoscienceAustralia/PyRate/pull/302/commits/fa2a65850f4aec5bfc26712c9339f19b6edc51ee